src/wazuh_modules/vulnerability_detector/wm_vuln_detector.c

```diff
@@ -5465,6 +5470,17 @@ int wm_vuldet_collect_agents_to_scan(wm_vuldet_t *vuldet) {
             os_strdup(agti_arch, agents->arch);
         }
 
+        // Correcting the achitecture string to generate a valid OS CPE
```

Comment:
Fix this typo: "achitecture" -> "architecture".

And fix the [code style](https://github.com/wazuh/wazuh/wiki/Coding-style-guide-(C)#style-conventions) (spaces) on the following lines, please.

Reply:
Fixed!
Thank you